botorch/models/multitask.py

```diff
@@ -403,6 +403,7 @@ def __init__(
         train_X: Tensor,
         train_Y: Tensor,
         likelihood: Optional[MultitaskGaussianLikelihood] = None,
+        data_covar_module: Optional[Module] = None,
```

Comment:
a little out of scope for the original intent of the PR, but it's a really helpful change that shouldn't break any tests